doc/languages-frameworks/gradle.section.md

```diff
@@ -0,0 +1,189 @@
+# Gradle {#gradle}
+
+Gradle is a popular build tool for Java/Kotlin. Gradle itself doesn't
+currently provide tools to make dependency resolution reproducible, so
+nixpkgs has a proxy designed for intercepting Gradle web requests to
+record dependencies so they can be restored in a reproducible fashion.
+
+## Building a Gradle package {#building-a-gradle-package}
+
+Here's how a typical derivation will look like:
+
+```nix
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
[`finalAttrs` are not used anymore](https://github.com/NixOS/nixpkgs/pull/326677#discussion_r1677222295), please adjust this

Reply:
Oh my bad